exmg-paper-combobox.ts

```diff
@@ -231,7 +278,7 @@ export class PaperComboboxElement extends LitElement {
     }
 
     const id = this.getSelectedItemKey(this.selectedItem);
-    if (typeof id === 'undefined') {
+    if (typeof id === 'undefined' || id === -1) {
```

Comment:
are you sure "-1" is not valid value? I think when "-1" you have to also check for attr-for-selected=true

Reply:
It was fine check as values from attributes are always string. However added more precise check